docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+
+*Please note that the above patch is not available for Windows Server version 2019.*
+
+> Comming soon: Support for Windows Server versions 2004 (build 19041.\*) and 20H2 (build 19042.\*) Semi-Annual Channels.
+
+
+## 2. Create the virtual machine
+
+To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
+See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html)
+
+
+### Setup VMware Tools
+
+Install VMware Tools version 11.0.6 or greater. See the [VMware Tools documentation](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-D8892B15-73A5-4FCE-AB7D-56C2C90BD951.html) for more information.
+
+Enable the VMware Tools configuration file [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) within the Windows virtual machine.
+
+For example in Windows Server version 1909 the configuration file is located at
+
+```powershell
+    C:\ProgramData\VMware\VMware Tools\tools.conf
+```
+
+In case the configuration file does not exist, VMware Tools installs an example configuration file in the same 
+directory as the location above or you can download it from the [open-vm-tools](https://raw.githubusercontent.com/vmware/open-vm-tools/master/open-vm-tools/tools.conf) repository.
+
+Ensure the virtual machine has a valid IP address in vCenter.
+
+Ensure the **VMTools** Windows service is running and will start on boot.
+
+
+### Exclude the network interface in the VMware Tools configuration
+
+Enable the following entry in the VMware Tools configuration file to ensure that the cloned vNIC generated on 
+the Windows VM by the hybrid-overlay won't be ignored and the virtual machine has a valid IP Address in vCenter
+
+```bash
+    exclude-nics=
+``` 
+
+In addition, you can run the following PowerShell script that downloads the example configuration file, excludes 
+the network interface and creates the `tools.conf` file.
+
+```powershell
+    Invoke-WebRequest -O tools.conf https://raw.githubusercontent.com/vmware/open-vm-tools/master/open-vm-tools/tools.conf
+    
+    (Get-Content -Path tools.conf) -replace '#exclude-nics=','exclude-nics=' | Set-Content -Force -Path tools.conf
+    
+    mv -Force tools.conf 'C:\ProgramData\VMware\VMware Tools\tools.conf'
+```
+
+> IPv6 is not supported and has to be disabled in the golden image.
+
+
+## 3. Setup SSH
+
+Install and configure the OpenSSH Server on the Windows virtual machine with automatic startup and key based-authentication.
+See [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse) to install using PowerShell.
+
+Ensure the SSH service is running and that it will start on boot.
+
+
+### Add the Administrator Public Key
```

Comment:
It is unclear from this paragraph how this key should be installed. You need to give the manual steps and the alternative is to use the script.